eval/asserts/fix-closure.mjs

```
@@ -46,21 +203,106 @@ export default async function assertFixClosure() {
       missingAttributionRejected = true;
     }
 
-    const hasAttribution = (event, expected) =>
-      event?.scope === expected.scope && event?.iteration === expected.iteration;
+    const pending = { provider: 'secondary', stepIteration: 3 };
+    const createAttemptState = () => ({ pending: { ...pending }, attempts: ['primary', 'secondary'] });
+    const successInput = createAttemptState();
+    const successSnapshot = structuredClone(successInput);
```

ESLint

[error] 'structuredClone' is not defined.

(no-undef)